test/utils/fizzy_engine.cpp

```diff
@@ -27,6 +28,31 @@ class FizzyEngine : public WasmEngine
 
 namespace
 {
+ValType translate_valtype(char input)
+{
+    if (input == 'i')
+        return fizzy::ValType::i32;
+    else if (input == 'I')
+        return fizzy::ValType::i64;
+    else
+        throw std::runtime_error{"invalid type"};
+}
+
+FuncType translate_signature(std::string_view signature)
+{
+    const auto delimiter_pos = signature.find(":");
+    assert(delimiter_pos != std::string_view::npos);
+    std::string_view inputs = signature.substr(0, delimiter_pos);
+    std::string_view outputs = signature.substr(delimiter_pos + 1);
+
+    FuncType func_type;
+    for (const auto input : inputs)
```

Comment:
This can be done as `std::transform(std::begin(input), std::end(input), std::back_inserter(func_type.inputs), translate_valtype)` or something...

Reply:
Sure, though I'm not convinced it is easier to read.

Reply:
Changed.